packages/light-apps/src/IdentityHeader/IdentityHeader.tsx

```diff
@@ -4,21 +4,34 @@
 
 import FileSaver from 'file-saver';
 import { AppContext, AlertsContext } from '@substrate/ui-common';
-import { Balance, CopyButton, Dropdown, FadedText, Icon, Input, Margin, Menu, Modal, NavLink, Stacked, StackedHorizontal, StyledLinkButton, WithSpaceAround, WithSpaceBetween, SubHeader } from '@substrate/ui-components';
+import uiSettings from '@polkadot/ui-settings';
+import { Balance, CopyButton, Dropdown, DropdownProps, FadedText, Icon, Input, Margin, Menu, Modal, NavLink, Stacked, StackedHorizontal, StyledLinkButton, WithSpaceAround, WithSpaceBetween, SubHeader } from '@substrate/ui-components';
 import React, { useContext, useState } from 'react';
 import { Route, RouteComponentProps, Switch } from 'react-router-dom';
 
 import { InputAddress } from './IdentityHeader.styles';
 
+const KEY_PREFIX = '__dropdown_option_';
+
 interface MatchParams {
   currentAccount: string;
 }
 
 interface Props extends RouteComponentProps<MatchParams> { }
 
+const nodeOptions: Array<any> = [];
+
+uiSettings.availableNodes.forEach(availNode => {
+  nodeOptions.push({
+    key: `${KEY_PREFIX}${nodeOptions.length}`,
```

Comment:
I think key should ideally not be equal to index, we could put availNode.value as key